### PR TITLE
[SW-6] Implement SchemaWizard runner

### PR DESCRIPTION
**Pull request by AndriyBosik**

### CODE REVIEW

**Review by NikolayProkopchuk — APPROVED** [review]

File: schema-wizard-core/src/main/java/org/schemawizard/core/dao/impl/TransactionServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                throw new MigrationApplicationException(sqlException.getMessage(), sqlException);
+            }
+            throw new MigrationApplicationException(exception.getMessage(), exception);
+        } finally {
+            try {
+                connection.close();
+            } catch (SQLException sqlException) {
+                log.warn("Unable to close database connection", sqlException);
+            }
+        }
```
Comment (ivlytvynets) [inline schema-wizard-core/src/main/java/org/schemawizard/core/dao/impl/TransactionServiceImpl.java:43]: try with resources, move to separate private method

File: schema-wizard-core/src/main/java/org/schemawizard/core/migration/service/impl/OperationResolverServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                .flatMap(List::stream)
+                .collect(Collectors.toMap(
+                        AbstractMap.SimpleEntry::getKey,
+                        AbstractMap.SimpleEntry::getValue,
+                        (first, second) -> {
+                            throw new ConflictDependencyException(String.format(
+                                    ErrorMessage.MULTIPLE_OPERATION_RESOLVERS_FORMAT,
+                                    first.getClass(),
+                                    second.getClass()));
+                        }));
```
Comment (ivlytvynets) [inline schema-wizard-core/src/main/java/org/schemawizard/core/migration/service/impl/OperationResolverServiceImpl.java:47]: move into init() method to improve visibility

**Review by ivlytvynets — CHANGES_REQUESTED** [review]

File: schema-wizard-core/src/main/java/org/schemawizard/core/dao/impl/TransactionServiceImpl.java (hunk 1 above)
Comment (AndriyBosik) [reply schema-wizard-core/src/main/java/org/schemawizard/core/dao/impl/TransactionServiceImpl.java:43]: unable to use try with resources
image

Comment (AndriyBosik) [reply schema-wizard-core/src/main/java/org/schemawizard/core/dao/impl/TransactionServiceImpl.java]: moved to separate method

File: schema-wizard-core/src/main/java/org/schemawizard/core/migration/service/impl/OperationResolverServiceImpl.java (hunk 2 above)
Comment (AndriyBosik) [reply schema-wizard-core/src/main/java/org/schemawizard/core/migration/service/impl/OperationResolverServiceImpl.java]: extracted to another method

**Review by ivlytvynets — APPROVED** [review]

**Pull request merged**